src/components/editor-content/editor-content.tsx

```diff
@@ -534,13 +558,114 @@ const EditorContent: React.FC<Props> = () => {
     );
   };
 
+  /**
+  * Renders vacation days subtitle text
+  * 
+  * @param name name of subtitle text
+  * @param value value of subtitle text
+  * @param unspent if it's displaying unspent vacation days
+  * @param positiveValue if amount of unused vacation days is positive 
+  */
+  const renderVacationDaysSubtitleText = (name: string, value: number, unspent: boolean, positiveValue?: boolean) => {
+    const valueColor = positiveValue ? theme.palette.success.main : theme.palette.error.main;
+    const valueText = value;
+
+    return (
+      <>
+        <Typography
+          variant="h4"
+          style={{ marginLeft: theme.spacing(2) }}
+        >
+          { name }
+        </Typography>
+        <Typography
+          variant="h3"
+          style={{
+            color: unspent ? valueColor : undefined,
+            marginLeft: theme.spacing(1),
+            fontStyle: "italic"
+          }}
+        >
+          { Math.abs(valueText) }
+        </Typography>
+      </>
+    );
+  };
+
+  /**
+  * Renders vacation days summary
+  */
+  const renderVacationDaysSummary = () => (
+    <>
+      <Typography variant="h2">
+        { strings.editorContent.vacationDays }
+      </Typography>
+      <Box className={ classes.vacationDaysSubtitle }>
+        { renderVacationDaysSubtitleText(`${strings.editorContent.spentVacationDays}`, person!.spentVacations, false) }
+        { renderVacationDaysSubtitleText(`${person!.unspentVacations >= 0 ? strings.editorContent.unspentVacationDays : strings.editorContent.extraVacationDays}`,
+          person!.unspentVacations, true, person!.unspentVacations >= 0) }
+      </Box>
+    </>
+  );
+
+  /**
+   * Renders vacation days list per week
+   */
+  const renderVacationDaysList = () => {
+    if (!vacationDayList) {
+      return null;
+    }
+    
+    return vacationDayList?.map((entry: VacationWeekData) => {
+      return (
+        <List className={classes.vacationList}>
+          <Typography style={{ fontSize: "1.2em" }}>{ strings.editorContent.week + entry.weekNumber }</Typography>
+          {entry.vacationDays.map(oneDay => {
+            return (
+              <ListItem>{oneDay.day.toLocaleDateString("fi-FI")}</ListItem>
+            );
+          })}
+        </List>
+      );
+    });
+  };
```

Comment:
Remove unnecessary returns,

Reply:
Done. Although linter seems to require weird formatting of parenthesis